tests/clients/test_experiment.py

```diff
@@ -312,11 +337,18 @@ def test_tag_schema_dump():
     assert data == TAG_BODY
 
 
-def test_experiment_run_data_schema():
+@pytest.mark.parametrize(
+    "metric,metric_body",
+    [[METRIC, METRIC_BODY], [METRIC_WITHOUT_STEP, METRIC_WITHOUT_STEP_BODY]],
```

Comment:
I find having the argument sets as tuples aids readability and better reflects their having a well defined length:
```suggestion
    [(METRIC, METRIC_BODY), (METRIC_WITHOUT_STEP, METRIC_WITHOUT_STEP_BODY)],
```